girderformindlogger/api/v1/user.py

```diff
@@ -683,6 +683,12 @@ def getUsersDetails(self):
                required=False, dataType='boolean')
         .param('status', 'The account status (admin access required)',
                required=False, enum=('pending', 'enabled', 'disabled'))
+        .param(
+             'login',
+             'Deprecated. Do not use.',
```

Comment:
Why add a new, immediately deprecated parameter to a deprecated API call rather than creating a new call?

Reply:
It's just comment with no meaning. So do you want to create new endpoint?

Reply:
Because when I check this comment https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/243#issuecomment-572117060 , I think you are going to change the username by text input.
And I found that the project use `login` variable instead of username.
Do you want to add new variable `username` and new endpoint for it?

Reply:
> It's just comment with no meaning. So do you want to create new endpoint?

Probably, based on comparing the existing endpoints to the desired functionality. Also, I think a "comment with no meaning" is not adequate for a parameter description, and by "to a deprecated API call", I mean the whole `PUT /user/{id}` endpoint is already marked deprecated:
![Warning: deprecated](https://user-images.githubusercontent.com/5974438/72448336-d2cbbe00-3784-11ea-9253-c6c981ac12bf.png)

Reply:
> Because when I check this comment [#243 (comment)](https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/243#issuecomment-572117060) , I think you are going to change the username by text input.
> And I found that the project use `login` variable instead of username.
> Do you want to add new variable `username` and new endpoint for it?

I'm sorry if that comment is unclear. There I meant that the issue of #243 is _just_ for the API call, not for updating any interfaces. And `login` is the variable/parameter name for usernames. Where I've referred to usernames, I mean it in natural language, not as a variable name.